### PR TITLE
Add readme & release assets

### PR DESCRIPTION
**Pull request by NaveenKharwar**
Need suggestions for readme.txt so that I can complete the README.md file too.

### CODE REVIEW

**Comment by ajitbohra** [comment]
@buddhamaan need your eyes on this

File: woocommerce-stats-by-currency.php [hunk 1]
```diff
@@ -2,7 +2,7 @@
 /**
  * Plugin Name: WooCommerce Stats By Currency
  * Plugin URI: https://lubus.in
- * Description: Add multicurrency stats on dashboard.
+ * Description: WooCommerce Stats By Currency is WordPress plugin by Lubus that shows you your WooCommerce stats based on your location at your own WordPress Dashboard.
```
Comment (ajitbohra) [inline woocommerce-stats-by-currency.php]: let's drop the lubus reference and keep it simple and it seems redundant here!

**Review by ajitbohra — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,2 @@
 /vendor/
-/node_modules/
\ No newline at end of file
+/node_modules/
```
Comment (ajitbohra) [inline .gitignore]: this changes should be separate PR ok for now

Comment (NaveenKharwar) [reply .gitignore]: Okay

**Comment by prajapati-pravin** [comment]
> Need suggestions for readme.txt so that I can complete the README.md file too.

Replace the current 'Donate Link' with a Patreon link.

File: readme.txt [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+Contributors: lubus,ajitbohra, naveenkharwar
+Donate link: http://www.lubus.in
+Tags: woocommerce,stats
+Requires at least: 3.8
+Tested up to:  5.4
+Stable tag: 1.0.0
+License: GPLv3
+License URI: http://www.gnu.org/licenses/gpl-3.0.html
+
+Test Various WordPress Queries
```
Comment (ajitbohra) [inline readme.txt]: this needs to be changed !

**Comment by ajitbohra** [comment]
> > Need suggestions for readme.txt so that I can complete the README.md file too.
> 
> Replace the current 'Donate Link' with a Patreon link.

@NaveenKharwar link https://www.patreon.com/lubus

**Review by ajitbohra — APPROVED** [review]

**Comment by ajitbohra** [comment]
looks good to me @buddhamaan any suggestion or we are good to go!
